github/ci/prow-deploy/kustom/base/configs/current/plugins/plugins.yaml

```diff
@@ -743,6 +743,8 @@ label:
     'kubevirt':
     - allowed_users:
       - kubevirt-bot
+      allowed_teams:
+      - needs-approver-review-label
```

Comment:
nit: I'd rename the label to `needs-approver-review`, that is, trim the `-label` suffix (if it's not too late).

Reply:
Ah, I've been confused with the label name itself.
Makes sense, thank you!